### PR TITLE
Bjorncs/feature flag owners

### PR DESCRIPTION
**Pull request by bjorncs**
I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

**Review by hakonhall — COMMENTED** [review]
there are some conflicts that needs to be resolved

File: config-model-api/src/main/java/com/yahoo/config/model/api/ModelContext.java [hunk 1]
```diff
@@ -50,8 +49,35 @@
...
+    /**
+     * How to remove temporary feature flags:
+     * 1)
+     * - Remove flag definition from Flags
+     * - Remove method implementation from ModelContextImpl.FeatureFlags
+     * - Modify default implementation of below method to return the new default value
+     * - Remove all usage of below method from config-model
+     *
+     * 2)
+     *  - Wait for all config-model versions in hosted production to include above changes
```
[the remark this answers is not in the document]
Comment (bjorncs) [reply config-model-api/src/main/java/com/yahoo/config/model/api/ModelContext.java]: Good idea 👍  See https://github.com/vespa-engine/vespa/pull/15639/commits/1ffe5453a05bc650c313885fe1a743010978cea4 for an implementation of your proposal.

**Comment by bjorncs** [comment]
Rebased with master to resolve merge conflict. The last commit is new.

**Comment by bjorncs** [comment]
@hakonhall PTAL

**Review by hakonhall — APPROVED** [review]

File: flags/src/main/java/com/yahoo/vespa/flags/Flags.java [hunk 2]
```diff
@@ -46,207 +42,133 @@
...
             "use-alternative-endpoint-certificate-provider", false,
-            List.of("nobody"), "2020-12-02", "2021-02-01",
+            List.of("andreer"), "2020-12-02", "2021-02-01",
             "Whether to use an alternative CA when provisioning new certificates",
             "Takes effect only on initial application deployment - not on later certificate refreshes!");
 
     public static final UnboundStringFlag DOCKER_IMAGE_REPO = defineStringFlag(
             "docker-image-repo", "",
-            List.of("nobody"), "2020-12-02", "2021-02-01",
+            List.of("valerijf"), "2020-12-02", "2021-02-01",
```
Comment (freva) [inline flags/src/main/java/com/yahoo/vespa/flags/Flags.java:171]: This was marked as permanent in the speadsheet...

Comment (bjorncs) [reply flags/src/main/java/com/yahoo/vespa/flags/Flags.java:171]: Thanks for noticing, I'll move in next PR 👍

**Pull request merged**